rpcd/playbooks/verify-maas.yml

```diff
@@ -18,9 +18,12 @@
   user: root
   tasks:
     - name: "Verify Checks & Alarms are registered"
-      script: >
-        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created
-        --entity {{ inventory_hostname }}
+      shell: |
+        {% if maas_venv_enabled | bool %}
+        . {{ maas_venv_bin }}/activate
+        {% endif %}
+        {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
```

Comment:
This could more simply be:

```
{{ maas_venv_bin }}/python {{ maas_rpc_scripts_dir }}/rpc-maas-tool.py verify-created \
-- entity {{ inventory_hostname }}
```

So you don't need to activate the virtualenv